calico-cloud_versioned_docs/version-3.16/image-assurance/scanners/pipeline-scanner.mdx

```diff
@@ -1,40 +1,47 @@
 ---
-description: Scan images using CLI scanner.
+description: Integrate the Image Assurance scanner into your build pipeline.
 ---
 
-# Scan images using the CLI scanner
+# Integrate Image Assurance scanner in your build pipeline
 
 ## Big picture
 
-Scan images manually or via CI/CD pipeline for vulnerabilities using {{prodname}} Image Assurance CLI scanner.
+Integrate the Image Assurance scanner into your build pipeline and private registries.
 
 ## Before you begin
 
 **Image requirements**
 
-To be included in scan results, images must:
+Images must:
 
-- Be available locally through the container runtime environment where the CLI scanner is running
+- Be available locally through the container runtime environment where the scanner is running
 - Have a RepoDigest ID
 
   To verify RepoDigestIDs, run this command:
 
   `docker images --digests`
 
-**CLI scanner requirements**
+**Scanner requirements**
 
 - Must have internet access to download and update the vulnerability database
-- To see image scan results in Manager UI, the CLI scanner must communicate with an external API endpoint outside your environment
+- To see image scan results in Manager UI, the scanner must communicate with an external API endpoint outside your environment
 
 ## How to
 
-- [Start the CLI scanner](#start-the-cli-scanner)
-- [Scan images using CLI](#scan-images-using-cli)
+- [Get latest version of Image Assurance](#get-latest-version-of-image-assurance)
+- [Start the scanner](#start-the-scanner)
+- [Scan images using the CLI](#scan-images-using-the-cli)
 - [Scan images using a config file](#scan-images-using-a-config-file)
+- [Get latest scanner updates](#get-latest-scanner-updates)
 
-### Start the CLI scanner
+### Get latest version of Image Assurance
 
-1. Download the latest version of the CLI scanner.
+1. On the **Managed Clusters** page, select the cluster from the list, and click **Reinstall**.
+1. Copy the updated installation script command and run it against your cluster.
```

Comment:
Hmm, I'm not sure if we need this. 

A user's managed cluster isn't directly linked with the CLI scanner. The CLI scanner talks to the Image Assurance API which is in the cloud.